main.tf

```diff
@@ -81,17 +55,24 @@ resource "azurerm_subnet" "subnet_for_each" {
   }
 }
 
-
 resource "azurerm_subnet_network_security_group_association" "vnet" {
```

Comment:
minor issue. Can we change the name of this resource to something meaningful like `subnet_association` than `vnet`. Same applies to other resources below which is named as `vnet`

Reply:
Resolved with b235fa7.